### PR TITLE
Issue489 fixflash

### PR DESCRIPTION
**Pull request by mdlpstsci**
Fixed a bug associated with the FLASHDUR computation.  The FLASHDUR keyword was updated in the CRJ/CRC headers to be the sum of the individual input images as this keyword is used in ACS2D to scale the flash reference file for flash correction.

### CODE REVIEW

File: pkg/acs/calacs/acsrej/acsrej_do.c [hunk 1]
```diff
@@ -395,6 +399,7 @@ int acsrej_do (IRAFPointer tpin, char *outfile, char *mtype, clpar *par,
                    "Total sky level (electrons)");
         PutKeyFlt (sg.globalhdr, "REJ_RATE", (float)nrej / texpt,
                    "Cosmic ray impact rate (pixels/sec)");
+        PutKeyFlt (sg.globalhdr, "FLASHDUR", cumFlashDur, "");
```
Comment (stsci-hack) [inline pkg/acs/calacs/acsrej/acsrej_do.c:402]: This new keyword should have a comment to explain what it is; specifically, total/cumulative flash duration in seconds (or minutes or hours or mjd? ).

**Review by stsci-hack — CHANGES_REQUESTED** [review]
The new keyword should have some description when added to the header, just to be nice to the user.

File: pkg/acs/calacs/acsrej/acsrej_do.c (hunk 1 above)
Comment (stsci-hack) [reply pkg/acs/calacs/acsrej/acsrej_do.c:406]: That's fine.

**Review by stsci-hack — APPROVED** [review]
LGTM

**Comment by mdlpstsci** [comment]
Resolves Issue #489.

**Pull request merged**